python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -269,9 +276,25 @@ def _chat_message_content_to_dict(self, message: "ChatMessageContent") -> dict[s
 
     async def _send_chat_request(self, settings: OpenAIChatPromptExecutionSettings) -> list["ChatMessageContent"]:
         """Send the chat request."""
+        span = model_diagnostics.start_completion_activity(settings.ai_model_id, MODEL_PROVIDER_NAME,
+                                                           self._settings_messages_to_prompt(settings.messages),
+                                                           settings)
+
         response = await self._send_request(request_settings=settings)
```

Comment:
What happens if there is an error / exception here? Shouldn't we be tracking errors in the telemetry?

Reply:
Addressed in coming iteration